core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/UserImpl.java

```diff
@@ -54,7 +57,7 @@
  * @since 0.0.2
  */
 @Entity
-@Table(name = "User")
+@Table(name = "`User`")
```

Comment:
I'm not entirely sure. It seems to work in some queries and not others. I'd rather leave it like this to be sure.